gp-includes/things/translation.php

```diff
@@ -299,7 +299,7 @@ public function set_fields( $db_object ) {
 	}
 
 	public function for_export( $project, $translation_set, $filters =  null ) {
```

Comment:
The default of `$filters` should probably be changed to an array now since that's the expected type by `for_translation()`.